docs/configuration.md

```diff
@@ -59,6 +62,19 @@ reports:
   - uri: https://salus-config.internal.net/salus-report
     format: json
     verbose: true
+  - uri: https://salus-config.internal2.net/salus-report
+    format: json
+    verbose: true
+    params:
+      repo: 'Random Repo'
+      user: 'John Doe' 
+  - uri: https://salus-config.internal2.net/salus-report
+    format: json
+    verbose: true
+    params:
+      repo: 'Random Repo'
+      user: 'John Doe' 
+      report: null
```

Comment:
On small tweak here.  To prevent a hard coded param name for the report itself we could just create a dedicated yaml config for that and then params could represent any additional ad hoc params to include in the POST

```
    salus_report_param_name: report
    additional_params:
      repo: 'Random Repo'
      user: 'John Doe' 
```

Reply:
but I'm not married to the names there, just the idea that we let the user decide the field names including the report one 👍